lenet/index.html

```diff
@@ -0,0 +1,118 @@
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>WebNN LeNet Example</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">   
+    <style>
+		.radioInit{
+			display: none;
+		}
+		.radioSelected{
+			display: inline-block;
```

Comment:
it would good to sort the css attributes alphabetically.

Reply:
good point. I'll fix that.

Reply:
Fixed. Actually I found these css attributes are unused, so I removed them.